src/main/java/ch/njol/skript/effects/EffConditional.java

```diff
@@ -0,0 +1,59 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.data.type.CommandBlock;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Conditional / Unconditional")
+@Description(
+	"Sets whether the provided command blocks are conditional or not."
+)
+@Examples({
+	"make {_block} conditional",
+	"make {_block} unconditional if {_block} is conditional"
+})
+@Since("INSERT VERSION")
+public class EffConditional extends Effect {
```

Comment:
This class name is very generic, when the actual effect is very specific. Best to be more specific that it's for command blocks imo